src/components/Card/Card.js

```diff
@@ -99,5 +109,17 @@ Card.propTypes = {
   bodyTextTemp: PropTypes.string
 }
 
+const mapStateToProps = state => {    
+  return {      
```

Comment:
excess return statement, in such simple arrow functions you can skip it
the same in other similar places